tests_async/test_credentials.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
similar, is this a test that is unique to async? If not we shouldn't duplicate it.

Reply:
These tests are unique to async, I changed the names of the test files so that this is clear. The import of credentials_async is renamed to credentials to make writing the tests shorter but we need to test the new async constructions.